### PR TITLE
feat: Designed and added landing page

### LINKED ISSUE #7: Landing Pages for Review App for SEO V2
### Create a SEO Friendly landing page for the Review app at `/landing`

As of now, This project has no landing pages or home pages. As you have already got an idea of what exactly this app does. It would be great if you could build a SEO friendly modern landing page with animations.

Here are some inspirations and Ideas
1. https://senja.io/
2. https://www.landingfolio.com/templates/post/saasfolio

> P.S - These links are only for inspiration, Please do not copy them exactly 😄

### PR DESCRIPTION
**Pull request by MeethDavda**
## What does this PR do?

This PR is to successfully add the Landing page along with animations and a visually appealing UI. Made necessary changes to improve the SEO of the page by adding meta tags which describe briefly what the application does and same with alt attributes 
of Image tags to help the SE understand the picture.
Fixes #7

Loom Video : https://www.loom.com/share/6270e54ee46749edb80755eabdb849d1?sid=4499a743-53c5-429f-9fe3-b72694a7d497

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- No unique test cases required.
- Reload the browser to see the initial animations.

## Mandatory Tasks

- [x] Self-reviewed the Code

### CODE REVIEW

**Comment by piyushgarg-dev** [comment]
Great Work 🎉

File: components/ui/footer.tsx
Comment (piyushgarg-dev) [inline components/ui/footer.tsx:1]: We need to change the footer as the links are invalid, such as Hosting, Meetup, Etc.
As of now let's remove all links and only keep @copyright one

File: components/ui/main.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        <h1 className="font-4 lh-6 ld-04 mb-6 text-center font-bold text-white lg:text-4xl">
+          Getting your customer reviews by only 1 Link
+        </h1>
+        <h2 className="font-4 lh-6 ld-04 pb-11 text-center font-semibold text-gray-700 lg:text-2xl">
+          Setting up a Review system can be quite tedious. Review App provides a
+          <br />
+          seamless and hassle free way to acquire feedback on your products.
+        </h2>
+        <div className="ml-6 flex flex-row items-center justify-center gap-10 text-center">
+          <div className="text-md focus:shadow-outline inline-flex transform animate-bounce items-center rounded-xl bg-transparent bg-white px-7 py-3 font-semibold text-black transition  md:mt-0">
```
Comment (piyushgarg-dev) [inline components/ui/main.tsx:19]: Remove the bounce effect

File: components/ui/main.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            <div className="flex  text-lg">
+              <span className=" justify-center text-purple-500">
+                Get Started
+              </span>
+            </div>
+          </div>
+          <div className="text-md  flex cursor-pointer items-center justify-center rounded-xl bg-transparent bg-gradient-to-r from-purple-500 to-purple-800 px-14 py-3 font-semibold tracking-tighter text-white  transition-all duration-200 ease-in-out hover:scale-125 md:mt-0">
+            <div className="flex text-center text-lg ">
```
Comment (piyushgarg-dev) [inline components/ui/main.tsx:27]: Make a signup to navigate to `app.localhost:3000/signup` and take these values from process.env

**Review by piyushgarg-dev — CHANGES_REQUESTED** [review]

**Comment by MeethDavda** [comment]
I have made the changes.

**Comment by piyushgarg-dev** [comment]
Hi @MeethDavda 
This webpage is not working in light mode, I needed to revert back this

**Comment by MeethDavda** [comment]
Actually the index page itself had some css applied before, which was in dark background. Should i add a theme change button ?

**Comment by NiKHiLkr23** [comment]
Hi @MeethDavda  @piyushgarg-dev  I see the pages aren't responsive, can I continue on this PR and add the specified theme and responsive changes ??

**Pull request merged**